packages/Babylonian-Core.package/BPExample.class/instance/assertions.st

```diff
@@ -0,0 +1,6 @@
+accessing
+assertions
+
+	^ self currentTrace
+		ifNil: [{}]
```

Comment:
The array is ideally a #(), as it is constructed at compile time, and does not allocate a new empty Array on every call. :)